boa/src/builtins/object/mod.rs

```diff
@@ -49,10 +50,11 @@ pub static PROTOTYPE: &str = "prototype";
 pub struct Object {
     /// The type of the object.
     pub data: ObjectData,
+    indexed_properties: FxHashMap<u32, Property>,
     /// Properties
-    properties: FxHashMap<RcString, Property>,
+    pub(crate) properties: FxHashMap<RcString, Property>,
```

Comment:
Hmmm I'm not sure about making this public (even if it's just in the crate). Could we maybe expose it with a method? Or, if we just use it to iterate, have a `properties_iter()` method that gets the `impl Iterator` directly?

Reply:
Ill remove the `pub(crate)` the `object.properties` stores string properties (key: `RcString`), we areay have an iterator for it `string_properties()`.